tests/sources/test_github.py

```diff
@@ -600,6 +600,18 @@ async def test_validate_config_with_invalid_token_then_raise():
             await source.validate_config()
 
 
+@pytest.mark.asyncio
+async def test_validate_config_with_extra_scopes_token():
+    async with create_github_source() as source:
+        source.github_client.post = AsyncMock(
+            return_value=(
+                {"user": "username"},
+                {"X-OAuth-Scopes": "user, repo, admin:org"},
+            )
+        )
+        assert await source.validate_config() is None
+
+
```

Comment:
Consider adding more test cases when `ConfigurableFieldValueError` is raised.

Reply:
added testcases